src/AbstractProxyFixer.php

```diff
@@ -53,6 +61,23 @@ public function getPriority()
     /**
      * {@inheritdoc}
      */
+    public function supports(\SplFileInfo $file)
+    {
+        return $this->proxyFixer->supports($file);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setWhitespacesConfig(WhitespacesFixerConfig $config)
+    {
+        parent::setWhitespacesConfig($config);
```

Comment:
I'm sorry I don't understand this.
If the real fixer needs a WhitespaceConfig (aka WC):

1. And the proxy isnt configuring it manually, real fixer setWC will raise an Exception
1. And the proxy configures it manually, everything is OK
1. And the proxy expects setWC to be called automatically and it doesn't implements WCAwareInterface, real fixer setWC will raise an Exception
1. And the proxy expects setWC to be called automatically and it implements WCAwareInterface, everything is OK
1. If in the tests someone calls proxy->setWC manually to make the tests pass without implementing WCAwareInterface, parent::setWC will raise an Exception

What is the case you talking of?